### ISSUE TITLE
"Retry and timeout are misconfigured" as per default settings

### ISSUE THREAD

**Issue by amo13** [Koed00/django-q#526]
On a fresh installation without altered configuration, the new warning pops up:
```
/usr/lib/python3.9/site-packages/django_q/conf.py:136: UserWarning: Retry and timeout are misconfigured. Set retry larger than timeout, 
        failure to do so will cause the tasks to be retriggered before completion. 
        See https://django-q.readthedocs.io/en/latest/configure.html#retry for details.
  warn("""Retry and timeout are misconfigured. Set retry larger than timeout,
```
Since I did not change any configuration, I looked up the default conf.py in this repo to find that timeout is set to none while retry is set to 60s. 
Should the the timeout in the default conf.py be set to an appropriate value so that it does not trigger the new warning after an unaltered default installation?

**Comment by Koed00** [comment]
Yes, and the docs need to be updated too. It happened because of a recent PR. Either that or we don't show the warning on no timeout configured.

**Comment by amo13** [comment]
Awesome! I can create the pull request, but I don't know what timeout value would be sane. 30? 59?

**Comment by mike-bell-gettogetherai** [comment]
I just had this issue on a fresh install with version 1.3.9 which is surprising since it looks like this PR was merged back in May...

**Comment by amo13** [comment]
It was not merged, only the modification in the documentation. I suggested the corresponding code modification in a separate pull request, but my code edits seem to create trouble elsewhere. Unfortunately, I do not know any python and cannot help any further.

#537

**Comment by mike-bell-gettogetherai** [comment]
Ah ok, thanks for confirming!

**Comment by seaturtlecode** [comment]
Is this warning safe to ignore if there is no timeout configured?

**Comment by abhijeet-electrum** [comment]
The same issue comes when we set the timeout value to `None`.  

How can we set timeout value to `None` and not get this warning? Tried keeping retry value as `None`, but same warning persist.

**Comment by sdarwin** [comment]
Hi,

I just encountered this error when installing something completely different, that happens to have django-q as a dependency.

Which means, at this moment, I'm not  designing a django-q system, per se.

There's a funny reddit https://www.reddit.com/r/explainlikeimfive/ "Explain Like I'm Five" which comes to mind.   In this Issue thread, could you explain in the most overly simplistic terms:  "If you encounter this error, change this settings, in this file" and then it's fixed. Done. For anyone who hits this error and ends up here.
